.github/workflows/build_and_push.yml

```diff
@@ -0,0 +1,55 @@
+name: Build and Push Container to ECR
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+
+env:
+  GITHUB_SHA: ${{ github.sha }}
+  REGISTRY: 545719101351.dkr.ecr.ca-central-1.amazonaws.com/next-auth-sandbox-wm
+
+permissions:
+  id-token: write
+  contents: read
+  pull-requests: write
+  actions: write
+  checks: write
+  statuses: write
+  security-events: write
+
+jobs:
+  build-push-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3.0.2
+
+      - name: Configure AWS credentials using OIDC
+        uses: aws-actions/configure-aws-credentials@8c3f20df09ac63af7b3ae3d7c91f105f857d8497 # v4.0.0
+        with:
+          role-to-assume: arn:aws:iam::414662622316:role/simplify-privacy-statements-V2-apply
+          role-session-name: ECRPush
+          aws-region: ca-central-1
```

Comment:
We don't usually have OIDC roles in Scratch accounts because of how they get cleaned out every week.  It would be easier to manage this if you AWS access keys in the account.